generators/entity-server/templates/partials/it_patch_update.partial.java.ejs

```diff
@@ -17,8 +17,10 @@
  limitations under the License.
 -%>
 // Initialize the database
-<%_ if (primaryKey.type === 'UUID' && databaseType !== 'sql') { _%>
+<%_ if (!primaryKey.autoGenerate && !databaseTypeSql) { _%>
+  <%_ if (primaryKey.typeUUID) { _%>
```

Comment:
We currently have 3 kinds of autoGenerate:
- `autoGenerate: false`
- `autoGenerate: true && liquibaseAutoIncrement: true` (generated by db)
- `autoGenerate: true && jpaGeneratedValue: true` (generated by jpa)

There are 2 problems:
- We need another type:
  - `autoGenerate: true && autoGenerateServerGenerated: true`, which is the default for nosql implementations. (generated by the service)
- primaryKey only exposes autoGenerate
  - need to use `primaryKey.fields[0].(autoGenerated|autoGenerateServerGenerated);
  - or loop over `primaryKey.fields`

The problem is that this test, uses the repository, skipping the service, which generates the id.
I can take a look at this separately.

Reply:
Yes please, I would like to do it the right way, but for now this primaryKey thing is kind of obscure to me...
If you find an elegant way to nested unescape, it would be very nice :-).
Or I can spend more time on it afterwards: as the behaviour is the same as before, merging this one won't hurt, then I can try to do my best to handle that remark for my next PR (and wait a little bit for constants)

Reply:
May this one work :-)?